.vscode/launch.json

```diff
@@ -272,6 +272,7 @@
                 "--no-timeouts",
                 "--exit",
                 "../../dist/test/${fileBasenameNoExtension}.js",
+                "../../../dist/test/**/${fileBasenameNoExtension}.js",
```

Comment:
does this also need to be in outdir? as the current outdir won't exist i don't think